interval/tools.go

```diff
@@ -8,3 +8,17 @@ import (
 func CoordsToString(i Interval) string {
 	return fmt.Sprintf("%s:%v-%v", i.GetChrom(), i.GetChromStart(), i.GetChromEnd())
 }
+
+func AreEqual(a Interval, b Interval) bool {
+	var isEqual bool = false
+
+	if a.GetChrom() == b.GetChrom() {
```

Comment:
Style: Would && be more efficient than the nested if checks?

Reply:
good point